com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -600,11 +600,49 @@ static void UpdateVolumeFramework(Camera camera, UniversalAdditionalCameraData a
                 UniversalAdditionalCameraData mainAdditionalCameraData = null;
 
                 if (mainCamera != null && mainCamera.TryGetComponent(out mainAdditionalCameraData))
+                {
                     layerMask = mainAdditionalCameraData.volumeLayerMask;
+                }
+
+                trigger = (mainAdditionalCameraData != null && mainAdditionalCameraData.volumeTrigger != null) ? mainAdditionalCameraData.volumeTrigger : trigger;
+            }
+
+            // We skip updating if the asset has volume updates disabled
+            if (additionalCameraData)
+            {
+                // Create stack for camera
+                if (additionalCameraData.volumeStack == null)
+                {
+                    additionalCameraData.volumeStack = VolumeManager.instance.CreateStack();
+                    VolumeManager.instance.Update(additionalCameraData.volumeStack, trigger, layerMask);
+                }
+
+                // When we have volume updates per-frame disabled...
+                if (!shouldUpdate)
+                {
+                    VolumeManager.instance.stack = additionalCameraData.volumeStack;
+
+                    // If the update setting was enabled in the previous scene, we need to update...
+                    if (additionalCameraData.lastRequiresVolumeFrameworkUpdate)
+                    {
+                        VolumeManager.instance.Update(additionalCameraData.volumeStack, trigger, layerMask);
+                        additionalCameraData.lastRequiresVolumeFrameworkUpdate = false;
+                    }
+
+                    // We always want to update normally in the editor when not in playmode
+                    #if UNITY_EDITOR
+                    if (Application.isPlaying)
+                    #endif
+                    {
+                        return;
```

Comment:
In the editor, outside playmode, we will update the volumes twice if `ViaScripting` is used. Once above and another time at the end of the function because we do not return here. We could probably restructure the two if statements here to only have one update.